quipucords/api/fingerprint/model.py

```diff
@@ -31,20 +31,36 @@ class SystemFingerprint(models.Model):
         ('unknown', 'Unknown')
     )
 
+    # Scan information
     fact_collection_id = models.ForeignKey(FactCollection,
                                            models.CASCADE)
     source_id = models.ForeignKey(Source, models.CASCADE)
     source_type = models.CharField(
         max_length=10, choices=SOURCE_TYPE)
 
+    # Common facts
     os_name = models.CharField(max_length=64, unique=False)
     os_release = models.CharField(max_length=128, unique=False)
     os_version = models.CharField(max_length=64, unique=False, null=True)
 
-    bios_uuid = models.CharField(max_length=36, unique=False)
-    subscription_manager_id = models.CharField(max_length=36, unique=False)
+    infrastructure_type = models.CharField(
+        max_length=10, choices=INFRASTRUCTURE_TYPE)
+    virtualized_is_guest = models.NullBooleanField()
+
+    mac_addresses = models.CharField(max_length=1024, unique=False, null=True)
+    ip_addresses = models.CharField(max_length=1024, unique=False, null=True)
 
     cpu_count = models.PositiveIntegerField(unique=False, null=True)
+
+    # Network scan facts
+    bios_uuid = models.CharField(max_length=36, unique=False, null=True)
+    subscription_manager_id = models.CharField(
+        max_length=36, unique=False, null=True)
+
+    connection_uuid = models.CharField(max_length=36, unique=False, null=True)
```

Comment:
I don't think we need the `connection_*` facts in the fingerprint.

Reply:
Done